distribution/docs/src/main/resources/_contents/configuring-from-the-admin-console-contents.adoc

```diff
@@ -1,6 +1,6 @@
 ===== Accessing the ${admin-console}
 
-* Open the admin portal. (Default: \https://localhost:8993/admin)
+* Navigate to ${admin-console}. (Default: \${secure_url}/admin)
```

Comment:
Do we need it to read: `Navigate to **the** ${admin-console}` or is the word **the** inserted for us automatically?